sass/directives/_visual-guides.scss

```diff
@@ -49,72 +53,119 @@
  * TODO: Concerning guides with no steps
  */
 @mixin visual-guide-row ($counter) {
-  @include display(flex);
+  display: block;
   position: relative;
 
-  .visual-guide__content, .visual-guide__image {
-    @include display(flex);
-    width: 50%;
+  &:focus {
+    outline: 0;
+  }
 
-    &:first-of-type {
-      margin-right: $base-spacing-large;
+  .visual-guide__image {
+    display: none;
+    
+    @media only screen and (min-width: $visual-guide-breakpoint-small) {
+      @include align-self(center);
+      @include justify-content(center);     
     }
+  }
+
+  .visual-guide__image--mobile {
+    margin-bottom: $base-spacing;
```

Comment:
@mrfantasticwonders FYI this is one of our old variables and it may be something we clean up next sprint. The new variables start here https://github.com/cb-talent-development/employer-style-base/blob/master/sass/variables/_sizing.scss#L52 but they're not solidified (there's nothing around 20px for instance) and we'll need to continue discussing with Susan what our consistent spacing should be